Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1

```diff
@@ -96,7 +96,7 @@ try
         Write-Verbose "Environment details $environment"
 
         Write-Verbose "Calling Deploy test agent cmdlet"
-        Invoke-DeployTestAgent -TaskContext $distributedTaskContext -MachineEnvironment $environment -UserName $machineUserName -Password $machinePassword -MachineNames $testMachineGroup -RunAsProcess $runAsProcess -LogonAutomatically $logonAutomatically -DisableScreenSaver $disableScreenSaver -AgentLocation $agentLocation -UpdateTestAgent $updateTestAgent -InstallAgentScriptLocation $installAgentScriptLocation -ConfigureTestAgentScriptLocation $configureTestAgentScriptLocation -CheckAgentInstallationScriptLocation $checkAgentInstallationScriptLocation -downloadTestAgentScriptLocation $downloadTestAgentScriptLocation -Connection $connection -PersonalAccessToken $personalAccessToken -DataCollectionOnly $isDataCollectionOnly -VerifyTestMachinesAreInUseScriptLocation $null
```

Comment:
if possible cleanup the if else above. We have added that to maintain compat. Id dont think it is necessary now. We can protect by changing minimum agent version